aten/src/ATen/native/native_functions.yaml

```diff
@@ -688,7 +688,7 @@
     SparseCUDA: bmm_sparse_cuda
   supports_named_tensor: True
 
-- func: _bmm(Tensor self, Tensor mat2, *, bool deterministic=False) -> Tensor
+- func: _bmm(Tensor self, Tensor mat2, *, bool? deterministic=None) -> Tensor
```

Comment:
Note that I made this change so that if the deterministic argument is not included in a `_bmm()` call, it will use `torch.experimental.deterministic`, rather than `False`